x-pack/test/cloud_security_posture_functional/cloud_tests/index.ts

```diff
@@ -10,6 +10,7 @@ import { FtrProviderContext } from '../ftr_provider_context';
 // eslint-disable-next-line import/no-default-export
 export default function ({ loadTestFile }: FtrProviderContext) {
   describe('Cloud Security Posture', function () {
+    loadTestFile(require.resolve('./agentless_sanity'));
```

Comment:
GHA workflow run is necessary because we are adding additional data generated by the agentless agent. It is important to verify that all suites continue to run as expected.